examples/pylab_examples/geo_demo.py

```diff
@@ -1,22 +1,19 @@
-import numpy as np
-#np.seterr("raise")
+import matplotlib.pyplot as plt
 
-from pylab import *
+plt.subplot(221, projection="aitoff")
```

Comment:
Could you replace this with the more explicit plt.subplot(2, 2, 1, ... and likewise below. The shorthand version is inherited from Matlab and I think we should encourage the more explicit version instead.